README.markdown

```diff
@@ -22,6 +24,9 @@ The projects follow the Liferay Workspace project style for gradle and maven.
 
 * `liferay-workspace` - in `gradle.properties`, Set `liferay.workspace.product`
   to the target product version.
+* `liferay-workspace-test-integration-sample` - Explain where the integration 
```

Comment:
@simonjhy , can you explain how users are supposed to execute the integration test?